app/src/main/res/navigation/graph_main.xml

```diff
@@ -187,6 +208,11 @@
                 android:id="@+id/action_locked_to_welcome"
                 app:destination="@id/fragment_welcome"
                 app:launchSingleTop="true"/>
+        <action
+                android:id="@+id/action_locked_to_filter"
+                app:destination="@id/fragment_filter_backdrop"
+                app:launchSingleTop="true"
+                app:popUpToInclusive="true"/>
```

Comment:
I think the intent of this issue was to share code between the two route presenters, not to merge the graph xml files.